exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -328,9 +432,11 @@
       //Iterates through all the nodes for update
       function reloadMetrics() {
           for (i = 1; i <= size; i++) {
-              var address = $("#row-"+i).find("#address").contents().get(0).nodeValue.trim();
-              var httpPort = $("#row-"+i).find("#httpPort").contents().get(0).nodeValue.trim();
-              updateMetricsHtml(address, httpPort, i);
+              if ( $("#row-"+i).find("#stateKey").length == 0 ) {
+                var address = $("#row-"+i).find("#address").contents().get(0).nodeValue.trim();
+                var httpPort = $("#row-"+i).find("#httpPort").contents().get(0).nodeValue.trim();
+                updateMetricsHtml(address, httpPort, i);
```

Comment:
Please update the check inside `updateMetricsHtml`, it will not refresh for local bit in case when Auth is enabled.
```
Current:
if ( !updateRemoteInfo || (location.protocol == "https" && remoteHost != location.host) ) {
           return;
}

Should Be:
if ( (!updateRemoteInfo || (location.protocol == "https") && remoteHost != location.host ) {
           return;
}
```

Reply:
Ok. Logically, this seems to resolve to the same outcome, though.

Reply:
Not true please see the braces. They evaluate to different outcome.